packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/UnionProperties.java

```diff
@@ -69,7 +69,7 @@ public Builder bar(software.amazon.jsii.tests.calculator.AllTypes bar) {
 
         /**
          * Sets the value of Foo
-         * @param foo the value to be set
+         * @param foo 
```

Comment:
We are losing something here. Bit of a pity.

Reply:
This was unintended as @bmaizels. Will add back in!